The -Wno-unused-result option is not understood by the version of gcc used by the
avr32 toolchain. Remove the option from the compilation flags.

Fixes build failures such as the following.

  http://autobuild.buildroot.net/results/e22/e22d94fca3eabb4e54d82af04319f17ad8e10c20/

Signed-off-by: Simon Dawson <spdawson@gmail.com>

diff -Nurp a/Makefile b/Makefile
--- a/Makefile	2013-11-21 17:13:17.679388374 +0000
+++ b/Makefile	2013-11-21 17:13:35.219299157 +0000
@@ -46,7 +46,7 @@ DEFINES = -DIGNORE_EMPTY_TEXT_NODES
 
 # options
 override CPPFLAGS += -Iinc/
-override CFLAGS += $(OPTIM) -fPIC -Wall -Wextra -Wno-unused-parameter -Wno-unused-result -Iinc/ $(DEFINES)
+override CFLAGS += $(OPTIM) -fPIC -Wall -Wextra -Wno-unused-parameter -Iinc/ $(DEFINES)
 override LDFLAGS += 
 
 ifeq ("$(OS)", "Darwin")
